modules/administration-guide/partials/proc_configuring-minikube-github-authentication.adoc

```diff
@@ -0,0 +1,57 @@
+[id="configuring-minikube-github-authentication_{context}"]
+= Configuring Minikube with GitHub Authentication
+
+
+Configure the OpenID Connect (OIDC) issuer to use GitHub authentication rather than preconfigured static users.
```

Comment:
```suggestion
Configure the {link-oidc-issuer} OpenID Connect (OIDC) issuer to use GitHub authentication rather than preconfigured static users.
```